### PR TITLE
Hints & Solution to 9 React and Redux Challenges

### PR DESCRIPTION
**Pull request by ShepherdNg**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md [hunk 1]
```diff
@@ -3,7 +3,18 @@ title: Connect Redux to React
...
+  <summary>Spoiler!</summary>
+  
+```jsx
+// change code below this line
+const ConnectedComponent = connect(mapStateToProps, mapDispatchToProps) (Presentational)
+```
+
+</details>
+
+<a href='https://github.com/freeCodeCamp/freeCodeCamp/blob/master/guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
```
Comment (thecodingaviator) [inline guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md]:
```suggestion
```
Remove this line and all lines like this from all the 9 files you're making changes to

File: guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md [hunk 2]
```diff
@@ -3,7 +3,18 @@ title: Connect Redux to React
...
 
 <a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
```
Comment (thecodingaviator) [inline guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md]:
```suggestion
```
Remove this line and all lines like this from all the 9 files you're making changes to

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
I see we have 10 challenges but you're adding only 9 solutions, also check comments below

**Comment by thecodingaviator** [comment]
Also, if you're making those changes, please pull the branch locally and squash commits

**Comment by ShepherdNg** [comment]
I just removed the links to github repo. Are they as desired now?
By the way, someone had provided solutions to the remaining challenge, so I just amended 9 files~
I am not familiar with how to "pull the branch locally and squash commits", hope it is still fine?

File: guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md (hunk 2 above)
Comment (ShepherdNg) [reply guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md]: I just removed the links to github repo. Are they as desired now?
By the way, someone had provided solutions to the remaining challenge, so I just amended 9 files~
I am not familiar with how to "pull the branch locally and squash commits", hope it is still fine?

Comment (thecodingaviator) [reply guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md]: It is fine, although, if you Google how to squash commits, it'll be easier

File: guide/english/machine-learning/deep-learning/index.md [hunk 3]
```diff
@@ -42,6 +42,10 @@ These days there are a variety of deep learning frameworks that allow you specif
 * <a href="https://github.com/gluon-api/gluon-api/">Gluon</a>
 * <a href="http://deeplearning.net/software/theano/">Theano</a>
 
+### Deep learning in Android & Web
```
Comment (thecodingaviator) [inline guide/english/machine-learning/deep-learning/index.md]: Please take these changes into PR, it seems to be out of scope of the PR

File: guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md (hunk 1 above)
Comment (thecodingaviator) [reply guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md]: LGTM

File: guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md (hunk 2 above)
Comment (thecodingaviator) [reply guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md]: LGTM

File: guide/english/machine-learning/deep-learning/index.md (hunk 3 above)
Comment (ShepherdNg) [reply guide/english/machine-learning/deep-learning/index.md]: Oops. I didn't touch this file. Just remove it?

Comment (thecodingaviator) [reply guide/english/machine-learning/deep-learning/index.md]: Yup, please remove it

Comment (RandellDawson) [reply guide/english/machine-learning/deep-learning/index.md]: @ShepherdNg Don't delete the file.  Just revert the changes for it.

**Comment by RandellDawson** [comment]
@thecodingaviator This PR has gone stale.  Also, I noticed a couple of the solutions did not include all the code (including challenge seed).  Also, I noticed the stub info was still present in at least 3 of the articles.  I attempted to clean everything up, but could not push to the user's repo.  I assume the user did not check the box to allow maintainers to make changes.  

I say close this PR and if the user does not ask for it to be reopened, I will create a PR with all the changes and fixes I made locally for this PR.

**Comment by thecodingaviator** [comment]
Alright @RandellDawson I'm closing this as stale. Please go ahead with your PR

**Review by thecodingaviator — COMMENTED** [review]
Hi,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️
